### PR TITLE
OCPBUGS-23013: Removing TP banner from kuryr migration docs.

### PR DESCRIPTION
**Pull request by gryf**
Version(s): 4.14.0, 4.15.0

Issue: https://issues.redhat.com/browse/OCPBUGS-23013

Link to docs preview: https://67477--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn

QE review:
- [ ] ~~QE has approved this change.~~

Additional information:

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@maxwelldb, PTAL.

File: networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc [hunk 1]
```diff
@@ -7,7 +7,6 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 :FeatureName: Migration from Kuryr to OVN-Kubernetes
-include::snippets/technology-preview.adoc[]
```
Comment (maxwelldb) [inline networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc:10]: You can delete L9, too--the snippet uses the `FeatureName` attribute.

**Review by maxwelldb — COMMENTED** [review]
One comment, @gryf.

**Comment by maxwelldb** [comment]
@kalexand-rh This shouldn't need change management, right? The RNs do properly indicate support status.

**Comment by kalexand-rh** [comment]
@maxwelldb, no, I think this is just a bug and doesn't need CM. @gryf, thank you for addressing this issue!

File: networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc (hunk 1 above)
Comment (gryf) [reply networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc:10]: ack.

**Comment by gryf** [comment]
@kalexand-rh yw!

**Review by maxwelldb — APPROVED** [review]

**Pull request merged**